README_maintainers.md

```diff
@@ -206,28 +206,20 @@ To remove all available node modules and start a clean installation run:
 
 The frontend can also be built using these commands:
 
-    cd <SourceRoot>/js/apps/system/_admin/aardvark/APP/
-    npm install
-    grunt deploy
+    cd <SourceRoot>/js/apps/system/_admin/aardvark/APP/react
+    npm run build
```

Comment:
don't we need to run `npm install` any more?
Assuming we have not all modules installed, but a fresh checkout.

Reply:
My mistake, need `npm install` still of course.

Reply:
fixed.